examples/flutter_gallery/test/demo/cupertino/cupertino_nagivation_demo_test.dart

```diff
@@ -0,0 +1,49 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter_gallery/demo/cupertino/cupertino_navigation_demo.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/foundation.dart';
+
+void main() {
+
+  // This verifies that specifically on android devices, a page back
+  // swipe doesn't work when looking at Cupertino pages in the flutter gallery.
+  testWidgets('test only edge swipes work (LTR)', (WidgetTester tester) async {
```

Comment:
Since we are specifically changing the gallery I think we should actually add the test for iOS.